tests/tests.rs

```diff
@@ -2,6 +2,6 @@ use goldentests::{ TestConfig, TestResult };
 
 #[test]
 fn run_goldentests() -> TestResult<()> {
-    let config = TestConfig::new("python", "examples", "# ");
+    let config = TestConfig::new("python", "examples", "# ")?;
```

Comment:
Nice :+1:

Reply:
Ah actually, forgot one thing related to this change. I think the documentation would need to be updated to add the extra `?`. Pretty minor but I don't want people to get confused. You should be able to find it by grepping for `TestConfig::new(` I think there are 2-3 instances of it?